### PR TITLE
Add more UI tests

### PR DESCRIPTION
**Pull request by antonrohr**
2 tests for ConnectToStadiaWidget and SessionSetupDialog that are both rather smokey. To add more correctness tests, more refactoring is necessary. For now its better to have these tests than not having them.

### CODE REVIEW

File: src/SessionSetup/ConnectToStadiaWidgetTest.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#include <QTableView>
+
+#include "SessionSetup/ConnectToStadiaWidget.h"
+
+namespace orbit_session_setup {
+
+TEST(ConnectToStadiaWidget, IsSetActive) {
+  ConnectToStadiaWidget widget{};
+
+  auto* radio_button = widget.findChild<QRadioButton*>("radioButton");
```
Comment (florian-kuebler) [inline src/SessionSetup/ConnectToStadiaWidgetTest.cpp:18]: This is the radio button to choose for "Connect to Stadia instance" right? There are others below for the file loading and local profiling, right? Maybe add a very brief comment here?

File: src/SessionSetup/SessionSetupDialogTest.cpp [hunk 2]
```diff
@@ -6,38 +6,81 @@
...
 
 namespace orbit_session_setup {
 
-// TODO(b/189838600) Add tests that check correctness.
+namespace {
+orbit_metrics_uploader::MetricsUploaderStub uploader_stub{};
+}
 
-TEST(SessionSetupDialog, SmokeTest) {
+TEST(SessionSetupDialog, Exec) {
```
Comment (florian-kuebler) [inline src/SessionSetup/SessionSetupDialogTest.cpp:27]: I would prefer a more descriptive test name. Test names can be quite large, but ideally they should describe what gets tested and what is the expected output. 

In this case I would also split the test into two separate tests, CreateExecAndRejectEmptyDialogueReturnsNoConfiguration and CreateExecAndStartDialogueWithFileTargetReturnsValidConfiguration.
This goes a bit more into the direction of "behaviour tests".

File: src/SessionSetup/ConnectToStadiaWidgetTest.cpp (hunk 1 above)
Comment (antonrohr) [reply src/SessionSetup/ConnectToStadiaWidgetTest.cpp:21]: This is just the ConnectToStadiaWidget, so its always the same radio button. I added a more clarifying comment. ptal

File: src/SessionSetup/SessionSetupDialogTest.cpp (hunk 2 above)
Comment (antonrohr) [reply src/SessionSetup/SessionSetupDialogTest.cpp]: done :)

Comment (antonrohr) [reply src/SessionSetup/SessionSetupDialogTest.cpp]: I actually also added a test fixture to reuse code.. hence the fixup change has plenty changes

File: src/SessionSetup/ConnectToStadiaWidgetTest.cpp (hunk 1 above)
Comment (florian-kuebler) [reply src/SessionSetup/ConnectToStadiaWidgetTest.cpp:21]: Ah thanks!

**Review by florian-kuebler — APPROVED** [review]
LGTM, thanks!

File: src/SessionSetup/SessionSetupDialogTest.cpp (hunk 2 above)
Comment (florian-kuebler) [reply src/SessionSetup/SessionSetupDialogTest.cpp]: Thanks! I had another look at the complete PR. LGTM.

**Pull request merged**